R/plot_ci.R

```diff
@@ -12,68 +12,72 @@
 #' @param bin_size A integer vector for number of bins representing intervals of
 #'                 equal size over the range
 #' @param n A integer or character vector for the size of bootstrap samples
-#' @param ci_level A numeric vector for the confidence level
-#' @param ci_random_seed A integer vector as random seed. 
+#' @param level A numeric vector for the confidence level
+#' @param random_seed A integer vector as random seed. 
 #' @param title A character vector for the title of the histogram
-#' @param x_axis A character vector for the name of the x axis
 #' @param y_axis A character vector for the name of the y axis
-#' @param save_result_to A character vector for the path to directory from 
-#'                       current directory to save plot
+#' @param estimator A character vector containing one of the("mean", "median", 
+#'                  "var", "sd") estimators
+#' @param path A character vector for the path to directory from 
+#'             current directory to save plot
 #'
 #' @return ggplot object
 #' @export
 #'
 #' @examples
-#' plot_ci(c(1, 2, 3, 4, 5, 6, 7), 1000, n = 100, ci_level = 0.95)
-#' plot_ci(c(1, 2, 3, 4, 5, 6, 7), 1000, n = 100, save_result_to = "../")
+#' plot_ci(c(1, 2, 3, 4, 5, 6, 7), 1000, n = 100, level = 0.95)
+#' plot_ci(c(1, 2, 3, 4, 5, 6, 7), 1000, n = 100, path = "../")
 
-# library(ggplot2, quietly = TRUE)
-
-plot_ci <- function(sample, rep, bin_size = 30, n = "auto", ci_level = 0.95, 
-                    ci_random_seed = NULL, title = "Bootstrap Sampling Plot", 
-                    x_axis = "Bootstrap Sample Mean", y_axis = "Count", 
-                    save_result_to = ".") {
+plot_ci <- function(sample, rep, bin_size = 30, n = "auto", level = 0.95, 
+                    random_seed = NULL, title = "", y_axis = "Count", 
```

Comment:
```suggestion
                    seed = NULL, title = "", y_axis = "Count", 
```
In R we can call it just seed. Thanks